src/hotspot/share/prims/universalUpcallHandler.cpp

```diff
@@ -60,7 +79,7 @@ void ProgrammableUpcallHandler::detach_current_thread() {
 
 // modelled after JavaCallWrapper::JavaCallWrapper
 JavaThread* ProgrammableUpcallHandler::on_entry(OptimizedEntryBlob::FrameData* context) {
-  JavaThread* thread = maybe_attach_and_get_thread(&context->should_detach);
```

Comment:
isn't that already the case in codeBlob.hpp ?

Reply:
Oh, huh. You're right. I think those changes might have been hidden when I looked at the changes, since I already looked at them before. (Now I see it).